scripts/docker-compose-bundled.yml.template

```diff
@@ -349,6 +349,174 @@ services:
+      image: itechuw/openelis-global-2-db-bundled:main
+      platform: linux/amd64
+      ports:
+          - "15432:5432"
+      restart: always
+      environment:
+        - POSTGRES_USER=postgres
+        - POSTGRES_PASSWORD=admin
+        - POSTGRES_DB=clinlims
+        - POSTGRES_INITDB_ARGS="--auth-host=md5"                  
+      networks:
+        - ozone
+      volumes:
+            - db-data:/var/lib/postgresql/data        
+      healthcheck:
+          test: [ "CMD", "pg_isready", "-q", "-d", "clinlims", "-U", "clinlims" ]
+          timeout: 45s
+          interval: 10s
+          retries: 10 
+          
+  oe.openelis.org:
+      container_name: openelisglobal-webapp 
+      image: itechuw/openelis-global-2-bundled:main
+      platform: linux/amd64  
+      depends_on:
+          - db.openelis.org
+          - certs
+      ports:
+          - "8443:8443"
+      restart: always
+      networks:
+          - ozone
+      environment:
+          - DEFAULT_PW=adminADMIN! 
+          - TZ=America/New_York
+            # context.xml doesn't seem to be able to pick up environment variables directly, so we are passing them in as CATALINA_OPTS
+          - CATALINA_OPTS= -Ddatasource.url=jdbc:postgresql://db.openelis.org:5432/clinlims -Ddatasource.username=clinlims
+          # Remote server Configuration
+          - ORG_OPENELISGLOBAL_REMOTE_SOURCE_URI=http://fhir.openelis.org:8080/fhir/
+          - ORG_OPENELISGLOBAL_REMOTE_SOURCE_UPDATESTATUS=true
+          - ORG_OPENELISGLOBAL_REMOTE_SOURCE_IDENTIFIER=Practitioner/*
+          - ORG_OPENELISGLOBAL_REMOTE_POLL_FREQUENCY=120000
+          - ORG_OPENELISGLOBAL_TASK_USEBASEDON=true
+          # SAML Configuration
+          - ORG_ITECH_LOGIN_SAML=true
+          - ORG_ITECH_LOGIN_SAML_REGISTRATIONID=keycloak
+          - ORG_ITECH_LOGIN_SAML_ENTITYID=OpenELIS-Global_saml
+          - ORG_ITECH_LOGIN_SAML_METADATALOCATION=http://keycloak:8080/realms/OpenELIS/protocol/saml/descriptor
+
+      volumes:
+          -  key_trust-store-volume:/etc/openelis-global
+          -  lucene_index-vol:/var/lib/lucene_index
+          
+  fhir.openelis.org:
+      container_name: external-fhir-api
+      image: itechuw/openelis-global-2-fhir:latest
+      platform: linux/amd64
+      depends_on:
+          - db.openelis.org
+          - certs    
+      networks:
+          - ozone
+      restart: always
+      environment:
+          # --- MANAGEMENT / ACTUATOR ---
+          - management.endpoints.web.exposure.include=health,prometheus
+
+          # --- SPRING CONFIG ---
+          - spring.main.allow-circular-references=true
+          - spring.flyway.enabled=false
+          - spring.flyway.check-location=false
+          - spring.flyway.baselineOnMigrate=true
+
+          - spring.datasource.url=jdbc:postgresql://db.openelis.org:5432/clinlims?currentSchema=clinlims
+          - spring.datasource.username=clinlims
+          - spring.datasource.password=clinlims
```

Comment:
Let's maintain the ability to override this at deployment time by using env vars e.g 
```docker
spring.datasource.username=\${CLINLIMS_USERNAME}
```
then provide the default in `openmrs-his.env` file. Esp. for passwords and secrets